### DISCUSSION TITLE
Calling All Appwrite Users! Vote for Appwrite Integration in FlutterFlow!

### DISCUSSION THREAD

**Discussion by bhjvhjvhjvjh** [appwrite/appwrite#5841] [General]
As devoted Appwrite users, we now have the amazing opportunity to influence the integration of Appwrite with FlutterFlow. 

To cast your vote for Appwrite integration in FlutterFlow, follow these simple steps:

Head over to the FlutterFlow community forum: [Flutterflow Community
](https://community.flutterflow.io/)

Navigate to the "Wishlist" section. you need a flutterflow account to be able to vote.

Within the list, you'll find a voting poll. Be sure to select the option supporting Appwrite integration

as of right now Appwrite as 18 votes.

 The FlutterFlow team closely considers community feedback, and our votes carry significant weight in 

if this post is not the appropriate here  for such an announcement, let me know i will remove it.

**Comment by mauricev** [comment]
I just added the 104th vote.
